mobly/base_test.py

```diff
@@ -887,7 +887,7 @@ def _safe_exec_func(self, func, *args):
       return func(*args)
     except signals.TestAbortAll:
       raise
-    except:
+    except Exception:
```

Comment:
Fix according to Flake8 rule [E722](https://www.flake8rules.com/rules/E722.html)
```
Do not use bare except, specify exception instead
```